system/php-dom-wrapper/Traits/ManipulationTrait.php

```diff
@@ -220,6 +220,8 @@ public function setText($input) {
             // Add new contents in it's place.
             $node->append(new Text($newNodes->getText()));
         });
+
+        return $this;
```

Comment:
There is an inconsistency with the return type in the annotation and the actual return type. Please make it consistent.